Cabal-tests/tests/UnitTests/Distribution/Simple/Program/GHC.hs

```diff
@@ -190,35 +190,4 @@ options_9_0_all =
 options_9_0_affects :: [String]
 options_9_0_affects =
     [ "-fcmm-static-pred"
-    ]
-
--------------------------------------------------------------------------------
--- GHC-9.2
--------------------------------------------------------------------------------
-
-options_9_2_all :: [String]
-options_9_2_all =
-    [ "-dynohi"
-    , "-ddump-c-backend"
-    , "-ddump-stg-from-core"
-    , "-ddump-stg"
-    , "-ddump-faststrings"
-    , "--run"
-    , "-ffamily-application-cache"
-    , "-fno-family-application-cache"
-    ] ++ options_9_2_affects
-
-options_9_2_affects :: [String]
-options_9_2_affects =
-    [ "-fprof-callers"
-    , "-funfolding-case-threshold"
-    , "-funfolding-case-scaling"
-    , "-fdistinct-constructor-tables"
-    , "-finfo-table-map"
-    , "-fexpose-internal-symbols"
-    , "-finline-generics"
-    , "-finline-generics-aggressively"
-    , "-fno-expose-internal-symbols"
-    , "-fno-inline-generics"
-    , "-fno-inline-generics-aggressively"
```

Comment:
Why is this not a test similar to the tests for 8.8, 8.10 and 9.0 options?

Reply:
Yes, GHC option checking has slipped through the cracks, see #9435.